proposals/1406_multiple-idps/README.md

```diff
@@ -0,0 +1,493 @@
+         - name: onlyIncludeGroupsWithThisPrefix
+           type: string
+           stringValue: "kube/"
+         - name: mustBelongToOneOfThese
+           type: stringList
+           stringListValue: [ kube/admins, kube/developers, kube/auditors ]
+         - name: additionalAdmins
+           type: stringList
+           stringListValue: [ ryan@example.com, ben@example.com, josh@example.com ]
+
+       # An optional list of transforms and policies to be executed in the order given during every login attempt.
+       # Each is a CEL expression. It may use the basic CEL language plus the CEL string extensions from cel-go.
+       # The username, groups, and the constants defined above are available as variables in all expressions.
+       # In the first version of this feature, the only allowed types would be policy/v1, username/v1, and groups/v1.
+       # This leaves room for other future possible types and type versions.
+       # Each policy/v1 must return a boolean, and when it returns false, the login is rejected.
+       # Each username/v1 transform must return the new username (a string), which can be the same as the old username.
+       # Each groups/v1 transforms must return the new groups list (list of strings), which can be the same as the old
+       # groups list.
+       # After each expression, the new (potentially changed) username or groups get passed to the following expression.
+       # Any compilation or type-checking failure of any expression will cause an error status on the FederationDomain.
+       # Any unexpected runtime evaluation errors (e.g. division by zero) cause the login to fail.
+       # When all expressions evaluate successfully, then the username and groups has been decided for that login.
+       expressions:
+         # This expression runs first, so it operates on unmodified usernames and groups as extracted from the IDP.
+         # It rejects auth for any user who does not belong to certain groups.
+         - type: policy/v1
+           expression: 'groups.exists(g, g in strListConst.mustBelongToOneOfThese)'
+           message: "Only users in certain kube groups are allowed to authenticate"
+         # This expression runs second, and the previous expression was a policy (which cannot change username or
+         # groups), so this expression also operates on the unmodified usernames and groups as extracted from the
+         # IDP. For certain users, this adds a new group to the list of groups.
+         - type: groups/v1
+           expression: 'username in strListConst.additionalAdmins ? groups + ["kube/admins"] : groups'
+         # This expression runs next. Due to the expression above, this expression operates on the original username,
+         # and on a potentially changed list of groups. This drops all groups which do not start with a certain prefix.
+         - type: groups/v1
+           expression: 'groups.filter(group, group.startsWith(onlyIncludeGroupsWithThisPrefix))'
```

Comment:
This example expression has a typo. `onlyIncludeGroupsWithThisPrefix` must be `strConst.onlyIncludeGroupsWithThisPrefix`.

Reply:
Fixed and force pushed.